tests/manage/test_create_storage_class_pvc.py

```diff
@@ -1,135 +1,133 @@
 import logging
-
 import pytest
 
-from ocs_ci.ocs import ocp, constants
-from ocs_ci.framework import config
+from ocs_ci.ocs import constants
 from ocs_ci.framework.testlib import tier1, ManageTest
-from ocs_ci.ocs.resources.ocs import OCS
-from ocs_ci.ocs.resources.pod import get_admin_key_from_ceph_tools
-from ocs_ci.ocs.resources.pvc import PVC
 from tests import helpers
-from ocs_ci.utility import templating
 
 log = logging.getLogger(__name__)
 
 
-POD = ocp.OCP(kind='Pod', namespace=config.ENV_DATA['cluster_namespace'])
-CEPH_OBJ = None
+@pytest.fixture(scope='function')
+def test_fixture_rbd(request):
+    def finalizer():
+        teardown_rbd()
+
+    request.addfinalizer(finalizer)
```

Comment:
You can add `teardown_fbd` as the finalizer directly here, no need to wrap it in the `finalizer` function.
```suggestion
    request.addfinalizer(teardown_rbd)
```

Reply:
Thanks, added in the script